views/newPatient.ejs

```diff
@@ -47,16 +47,16 @@
         <div class="rf-my-3w">
           <!--todo : little red star for required fields -->
           <label class="rf-label" for="firstnames">Prénoms</label>
-          <input class="rf-input midlength-input" type="text" id="firstnames" name="firstnames" required>
+          <input class="rf-input midlength-input" type="text" id="firstNames" name="firstNames" required>
```

Comment:
Il faut que le `for` aie la meme valeur que le `id` et le `name` pour qu'on aie pas d'emmerdes :
```suggestion
          <input class="rf-input midlength-input" type="text" id="firstnames" name="firstnames" required>
```